README.md

```diff
@@ -1 +1,49 @@
-# DTLib
\ No newline at end of file
+﻿# DTLib
+DTLib is an extension of [WPILib](https://github.com/wpilibsuite/allwpilib) designed by FRC Team 1559 for the [FIRST Robotics Competition](http://firstinspires.org/robotics/frc). It adds features that we felt were lacking, as well as improving code reusability and maintainability. The primary motivation behind this project is to ensure the software team can get up and running before the robot is fully assembled (or even designed).
+
+## License
+
+This library is licensed under the [GNU Public License V3](https://github.com/Team1559/DTLib/blob/master/LICENSE). However, not all dependencies are licensed in the same way. Notably, WPILib uses their own custom license (which can be found [here](https://github.com/wpilibsuite/allwpilib/blob/main/LICENSE.md)). To summarize, the licenses state that users are free to copy, modify and redistribute our code as much as they like, so long as we and our dependencies receive attribution for our work with the original licenses and copyrights included.
```

Comment:
You should avoid summarizing the license terms and let the license do that (GPL imposes requirements beyond the ones stated here)

Reply:
Got it, will leave out the summary. I think it’s still important to mention licenses for allwpilib and the vendor libraries.